<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Grapheur</class>
 <widget class="QMainWindow" name="Grapheur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grapheur</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblFreq">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Fr√©quence :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtFreq">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1000</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblBruit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Amplitude bruit :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtBruit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0.1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btValider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Valider</string>
    </property>
   </widget>
   <widget class="MplWidget" name="wdGraph" native="true">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>461</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btQuitter">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>748</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btQuitter</sender>
   <signal>clicked()</signal>
   <receiver>Grapheur</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
